Tasks/Kubernetes/Tests/L0.ts

```diff
@@ -311,6 +315,84 @@ describe('Kubernetes Suite', function() {
         console.log(tr.stderr);
         done();
     });
+ 
+    it('Runs successfully for kubectl create configMap from file or directory with forceUpdate', (done:MochaDone) => {
```

Comment:
Add test cases to validate following scenerios.
1. create secret -> create configmap -> kubectl commad execution. All three commands are getting executed.
2. create configmap -> kubectl commad. These two commands are independently getting executed.
3. create secret -> kubectl commad. These two commands are independently getting executed.